mchf-eclipse/drivers/audio/cw/cw_gen.c

```diff
@@ -226,7 +226,7 @@ static const float sm_table[CW_SMOOTH_TBL_SIZE] =
 
 void CwGen_SetSpeed()
 {
-    ps.dit_time         = 1650/ts.keyer_speed;      //100;
+    ps.dit_time         = 1920/ts.keyer_speed;      //100;
 }
```

Comment:
This is what I measure with the 1920 value:
6 WPM: 208 ms pause: 224 ms
12 WPM: 100 ms pause: 115 ms
24 WPM: 48 ms pause: 61.6 ms
48 WPM: 21.3 ms pause: 35.2 ms

So, still some work to do here to get the timings right.